clients/admin-ui/src/features/privacy-experience/preview/Preview.tsx

```diff
@@ -158,6 +162,18 @@ const Preview = ({
       : null;
   }, [initialValues, noticesOnConfig, values.component]);
 
+  useEffect(() => {
+    if (vendorCount === 0 && values.component === ComponentType.TCF_OVERLAY) {
+      notification.warning({
+        message: "No vendors available",
+        description:
+          "The TCF overlay will not display correctly until GVL vendors are added.",
```

Comment:
2 nits on this copy: 
1. What does "correctly" mean? Now we've updated it to not show entirely right?
2. Will an average user know what a GVL vendor is and where to configure them?